tests/btb/tuning/test_gp.py

```diff
@@ -186,13 +186,33 @@ class TestGPEi(TestCase):
     # METHOD: _acquire(self, predictions)
     # VALIDATE:
     #     * return values according to the formula
+    #@patch('btb.tuning.gp.GP')
+    def test__acquire_less_than_rmin(self): #, gp_mock):
+        """Best Expected Improvement corresponds to the best prediction
+        , as less than r_min proposals made."""
+        # Set-up
+        tuner = GPEi(tuple(), r_minimum=2)
+        tuner.X = np.array([[]])
```

Comment:
are you trying to initialize an empty array of shape=(1, 0)? why not be explicit
```python
tuner.X = np.empty(shape=(1, 0))
```